Add Department

zielinse opened the pull request:
Add DepartmentDTO
Add App Class with Departments Data

zielinse committed:
First commit
Add DepartmentDTO
Add App Class with Departments Data

zielinse committed:
Added App Class

RafalRomanQW commented on src/AppClass.java:
```diff
@@ -0,0 +1,16 @@
+    public static void main(String[] args){
+
+        DepartmentDTO hr = new DepartmentDTO( "HR", "Human Resourscers");
+        hr.setFullName("Human Resourscers");
```
if u use Constructor why do u set the same data again with setter ??

RafalRomanQW commented on src/AppClass.java:
```diff
@@ -0,0 +1,16 @@
+import DTO.DepartmentDTO;
+
+public class AppClass {
```
maybe AppData will be better?

RafalRomanQW commented on src/AppClass.java:
```diff
@@ -0,0 +1,16 @@
+import DTO.DepartmentDTO;
+
+public class AppClass {
+    public static void main(String[] args){
```
Shouldnt be in main

zielinse committed:
Corrected AppClass -> AppData
Main threw out from AppData
Setters Cancelled
Added package utils
Added list of department's objects
Added list of occupations

zielinse committed:
Corrected AppClass -> AppData
Main threw out from AppData
Setters Cancelled
Added package utils
Added list of department's objects
